cogs/commands/summarise.py

```diff
@@ -85,19 +114,30 @@ async def dispatch_api(self, messages) -> Optional[str]:
             reply = clean(reply, "Apollo: ", "apollo: ", name)
         return reply
 
-    async def create_message(self, message_chain, prompt):
+    async def create_message(self, message_chain, prompt, ctx):
         # get initial prompt
         initial = prompt + "\n"
 
         # for each message, append it to the prompt as follows --- author : message \n
+        message_length = 0
         async for msg in message_chain:
             if CONFIG.AI_INCLUDE_NAMES and msg.author != self.bot.user:
-                initial += msg.author.name + ":" + msg.content + "\n"
-
+                message_length += len(msg.content.split())
+                initial += msg.author.name + ": " + msg.content + "\n"
         messages = [dict(role="system", content=initial)]
 
         return messages
 
 
+    def build_prompt(self, bullet_points, channel_name, response_size):
+
+        bullet_points = "Put it in bullet points for readability." if bullet_points else ""
+        prompt = f"""People yap too much, I don't want to read all of it. The topic is related to {channel_name}. In {response_size} words or less give me the gist of what is being said. {bullet_points} Note that the messages are in reverse chronological order:
```

Comment:
Can you add this to the config file too, so we can experiment slightly

I reckon this could do with a bit of prompt tuning, I don't really know what I'm talking about, but maybe something like [this one](https://www.reddit.com/r/ChatGPTPro/comments/13n55w7/highly_efficient_prompt_for_summarizing_gpt4/). At least something to make it waffle less and be more precise